src/style.css

```diff
@@ -2,0 +1,53 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
```

Comment:
The same applies here for the bold version of Avenir. Always provide a list of fallback fonts.